backend/module/eCampCore/src/Entity/Activity.php

```diff
@@ -167,32 +160,33 @@ public function removeActivityResponsible(ActivityResponsible $activityResponsib
     public function PrePersist() {
         parent::PrePersist();
 
-        if (null !== $this->getActivityType() && $this->getActivityContents()->isEmpty()) {
-            $this->createDefaultActivityContents();
-        }
-    }
-
-    /**
-     * Replicates the default content-type structure given by the ActivityType.
-     */
-    public function createDefaultActivityContents(ContentTypeStrategyProvider $contentTypeStrategyProvider = null) {
-        foreach ($this->getActivityType()->getActivityTypeContentTypes() as $activityTypeContentType) {
-            for ($idx = 0; $idx < $activityTypeContentType->getDefaultInstances(); ++$idx) {
-                /** @var ContentType $contentType */
-                $contentType = $activityTypeContentType->getContentType();
-                $contentTypeName = $contentType->getName().' ';
-                $contentTypeName .= str_pad($idx + 1, 2, '0');
-
-                $activityContent = new ActivityContent();
-                $activityContent->setContentType($contentType);
-                $activityContent->setInstanceName($contentTypeName);
-
-                if ($contentTypeStrategyProvider) {
-                    $activityContent->setContentTypeStrategyProvider($contentTypeStrategyProvider);
-                }
-
-                $this->addActivityContent($activityContent);
-            }
-        }
-    }
+        // TODO:
+        // createDefaultActivityContents()
+        // use ActivityCategory -> ContentTypeConfig
+
+        // if (null !== $this->getActivityType() && $this->getActivityContents()->isEmpty()) {
+        //     $this->createDefaultActivityContents();
+        // }
```

Comment:
Wann wird dieses TODO abgearbeitet? Falls nicht in diesem PR, eröffnest du ein Issue? Oder planst du einen Teil 3 PR?

Reply:
Verschoben in ActivityService